amida-tech/amida-react-skeleton-graphql
# [DEVOPS-522] Dockerfile basics.

### Related tickets:
DEVOPS-522.

### What's changing? Any breaking changes?
Gives a Dockerfile that produces an nginx deployment. 

### Manual test cases?
Run `docker build -t amida-react-skeleton-graphql .` and then `docker run -p 3000:80 -e REACT_APP_GRAPHQL_API_URL=http://localhost:4000/ amida-react-skeleton-graphql` 

Afterwards, go to `http://localhost:4000` and confirm the site is up.
